README.md

```diff
@@ -81,19 +82,43 @@ vars:
     quickbooks_union_databases: ['quickbooks_usa','quickbooks_canada'] # use this if the data is in different databases/projects but uses the same schema name
 ```
 
-### Customize the account ordering of your financial statement models.
+
+### Customize the cash flow types and account ordering of your cash flow statement
+IMPORTANT: It is very likely you will need to reconfigure your `cash_flow_type` to make sure your cash flow statement matches your specific use case. Please examine the following instructions.
+
+The current default numbering for ordinals and default cash flow types are set in [the `int_quickbooks__cash_flow_classifications`](https://github.com/fivetran/dbt_quickbooks/blob/main/models/intermediate/int_quickbooks__cash_flow_classifications.sql) model. It' based on best practices for cash flow statements in accounting. You can see these ordinals being implemented in the `quickbooks__cash_flow_classifications` model, then implemented in the `quickbooks__cash_flow_statement` model. The `cash_flow_type` value is assigned off of `account_class`, `account_name` or `account_type`, and the cash flow ordinal is assigned off of `cash_flow_type`.
+
+If you'd like to modify either of these configurations, take the following steps to configure the fields you'd like to modify:
+
+1) Import a csv with fields into root (not the dbt package) `seeds` folder, then configure your `cash_flow_statement_type_ordinal` variable in your `dbt_project.yml` to reference the seed file name. 
+- For example, if you created a seed file named `quickbooks_cash_flow_types_ordinals.csv`, then you would edit the `cash_flow_statement_type_ordinal` in your root `dbt_project.yml` as such.
+ 
+  ```yml
+  vars:
+     cash_flow_statement_type_ordinal: "{ ref('quickbooks_cash_flow_types_ordinals') }"
```

Comment:
```suggestion
     cash_flow_statement_type_ordinal: "{{ ref('quickbooks_cash_flow_types_ordinals') }}"
```

Reply:
Can you also update this for the `financial_statement_ordinal` instructions below.